libvast/vast/config.hpp.in

```diff
@@ -8,6 +8,7 @@
 #cmakedefine01 VAST_ENABLE_DEVELOPER_MODE
 #cmakedefine01 VAST_ENABLE_EXCEPTIONS
 #cmakedefine01 VAST_ENABLE_JEMALLOC
+#cmakedefine01 VAST_ENABLE_JOURNALD_LOGGING
```

Comment:
You still need to change from `#ifdef` to `#if` and from `#ifndef` to `#if !` wherever you're using this, otherwise this cannot work.

Reply:
Right, I forgot to push the latest version.